reader/src/main/java/org/kaazing/monitoring/reader/agrona/extension/CountersManagerEx.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2007-2014 Kaazing Corporation. All rights reserved.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.kaazing.monitoring.reader.agrona.extension;
+
+import uk.co.real_logic.agrona.concurrent.AtomicBuffer;
+import uk.co.real_logic.agrona.concurrent.CountersManager;
+
+/**
+ * Wrapper over Agrona's Counter Manager to extend its functionality
+ * This class was created to expose a public getValue method for a given id, since the Agrona
+ * standard implementation did not provide one 
+ */
+public class CountersManagerEx extends CountersManager {
```

Comment:
Does Agrona expect the buffer passed to the CountersManager is something callers keep a reference to in order to get data out?  It kind of looks like it does based on a quick reading of the code, but I was just curious what the expected usage pattern of CountersManager is.  Thanks.